src/realm/table.hpp

```diff
@@ -1442,6 +1461,125 @@ class Table {
     friend class Group;
 };
 
+/*****************************************************************************/
+
+template <class T>
+using ListRef = std::unique_ptr<Table::List<T>>;
```

Comment:
Ok, I was not aware of that convention. However I created these types so that we later could change the implementation to actually return a std::shared_ptr - like it is done for the LinkViewRef.